lib/lita/handlers/pagerduty_utility.rb

```diff
@@ -51,6 +51,13 @@ class PagerdutyUtility < Handler
         }
       )
 
+      route(
+        /^pager\s+me\s+(\w+)\s+(\d+)m?$/,
```

Comment:
This regex currently doesn't allow for the possibility of schedule names containing spaces. Would you mind modifying it?